redash/handlers/alerts.py

```diff
@@ -73,6 +79,12 @@ def post(self):
 
     @require_permission('list_alerts')
     def get(self):
+        self.record_event({
+            'action': 'view',
+            'timestamp': int(time.time()),
+            'object_id': 'alerts',
+            'object_type': 'api_call'
+        })
```

Comment:
Let's convert all the list actions, where you used `object_type`=`api_call` to:

`action` -> `list`
`object_type` -> `<the relevant resource type>` (in this case it will be `alert`, to be consistent with the other events for this type).